### ISSUE TITLE
Add /time

### ISSUE THREAD

**Issue by Bearinx** [WarzoneMC/Warzone#107]
Basically, a command like this would allow staff to set a time limit for the map, this would be useful for tournaments and just basic gameplay. (/time set {time limit})

**Comment by JustNoah** [comment]
Very nice suggestion **+1**

**Comment by DeternoMC** [comment]
I might work on a pr for this.

**Comment by Synconix** [comment]
Nice suggestion, think you should move it to TO-DO

**Comment by Bearinx** [comment]
@jorgeberrex isn't this basically what you are doing right now?

**Comment by jorgeberrex** [comment]
Not exactly. I'm adding a time limit system, which will enable us to add this feature.
